packages/cozy-harvest-lib/src/components/Routes.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react'
 import { Switch, Route, Redirect } from 'react-router'
-import { withStyles } from '@material-ui/core/styles'
+import { withStyles } from 'cozy-ui/transpiled/react/styles'
```

Comment:
nit: Put to line 4

Reply:
c'est dans un commit qui n'est pas censé être dans cette PR (commit d'upgrade de lib, cf https://github.com/cozy/cozy-libs/pull/1891) je fais l'impasse ici donc.